reconciletags/reconciletags_e2e_test.py

```diff
@@ -0,0 +1,92 @@
+"""End to end test for the tag reconciler."""
+
+import json
+import os
+import subprocess
+import sys
+import unittest
+
+
+class ReconciletagsE2eTest(unittest.TestCase):
+
+    _BUCKET = ''
+    _FILE_NAME = 'e2e_test.json'
+    _DIR = 'tiny_docker_image/'
+    _REPO = 'gcr.io/google-appengine-qa/e2etest'
+    _TAG = 'initial'
+    _TEST_JSON = """
```

Comment:
It might be simpler to do this with an actual python object that you then json.dumps inline.